src/conduit/datakit_conduit.ml

```diff
@@ -153,6 +161,14 @@ let accept_forever ~sandbox ~serviceid ~make_root url =
        if Astring.String.is_prefix ~affix:"\\\\" url then begin
          Log.info (fun f -> f "Accepting connections on named pipe %s" url);
          Named_pipe.accept_forever url (Unix.handle ~make_root)
+       end else if String.is_prefix ~affix:"fd:" url then begin
+         let i = String.with_range ~first:3 ~len:(String.length url - 3) url in
```

Comment:
(note: you don't need `~len` here; taking the rest of the string is the default)